gui/wxpython/core/gthread.py

```diff
@@ -93,27 +93,27 @@ def run(self):
             "onterminate": None,
         }
         while True:
-            requestId, args, kwds = self.requestQ.get()
+            requestId, args, kwargs = self.requestQ.get()
             for key in ("callable", "ondone", "userdata", "onterminate"):
-                if key in kwds:
-                    variables[key] = kwds[key]
-                    del kwds[key]
+                if key in kwargs:
+                    variables[key] = kwargs[key]
+                    del kwargs[key]
 
             ret = None
             exception = None
             time.sleep(0.01)
 
             self._terminate_evt = wxThdTerminate(
                 onterminate=variables["onterminate"],
-                kwds=kwds,
+                kwargs=kwargs,
```

Comment:
Note to self: check if wxThdTerminate uses kwds or kwargs as named argument

Reply:
The only places where objects are defined with `kwds` in wxWidgets/Phoenix are [here](https://github.com/wxWidgets/Phoenix/blob/master/wx/svg/_nanosvg.c#L2168-L2212). No place in wxWidgets/wxWidgets.

Reply:
Nevertheless, let's stay on the safe side and keep `kwds` on the wx side, although the change seemed to be reasonable.